### PR TITLE
Fixes a migration generator warning:

### PR DESCRIPTION
**Pull request by noijd**
The message was:

[WARNING] Could not load generator "generators/smailer/migration_generator". Error: uninitialized constant Smailer::Compatibility.

I just included the smailer module and increment the version to 0.7.6.

### CODE REVIEW

**Comment by mitio** [comment]
@noijd Thanks for trying to fix this!

Can you tell me which version of Rails are you testing this in?

File: lib/generators/smailer/migration_generator.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 require "rails/generators/active_record"
+require_relative '../../smailer'
```
Comment (mitio) [inline lib/generators/smailer/migration_generator.rb]: Also, here the `smailer` gem and folder should be in the load path already, so it should be possible to just `require 'smailer'`. Does that fix the problem for you?

**Comment by noijd** [comment]
Sorry, my pull request lacks a lot of information:
I was including it inside another engine, with Rails 4.1.8.

I have tried your suggestion and updated my master branch with it.

**Comment by mitio** [comment]
Looks good to me. I haven't tested it with Rails 4.x, though. There might be more issues. I will push a new gem version, but if you find more problems, let's separate version bumping code from compatibility fixes in the future.

Thanks a lot!

**Comment by mitio** [comment]
v0.7.6 is live.

**Comment by noijd** [comment]
Ok, I will split it on my next pull request ;).

**Pull request merged**